src/components/primitive/stats-card/stats-card.component.tsx

```diff
@@ -0,0 +1,17 @@
+import { Avatar, Text } from 'components/primitive';
+import { StatsComponentProps } from './stats-card.component.props';
+import { StatsContainer, CountContainer } from './stats-card.component.styles';
+
+export const StatsCard: React.FC<StatsComponentProps> = (props) => {
```

Comment:
Make sure this is exported in `index.ts`

Reply:
yeah, it's exported in index.ts file.